GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1227,18 +1227,12 @@ private void OnGridViewMouseDown(object sender, MouseEventArgs e)
             }
         }
 
+        [ContractAnnotation("=>false,spi:null")]
+        [ContractAnnotation("=>true,spi:notnull")]
         internal bool TryGetSuperProjectInfo(out SuperProjectInfo spi)
         {
-            if (_superprojectCurrentCheckout.Task.IsCompleted)
```

Comment:
This condition should have been negated. `CompletedOrDefault` has equivalent implementation.